e2e/e2e_test.go

```diff
@@ -288,61 +278,9 @@ func TestUpgradesMajorVersion19to20(t *testing.T) {
 }
 
 func TestUpgradesMajorVersion19_1To19_2(t *testing.T) {
```

Comment:
We should not have a hardcoded version upgrade test, this needs to be changed. In the near term, lets move this to testing upgrading from 19.2 to 20.1 but in the future this should be dynamic

Reply:
19.1 is no longer supported anyway